packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -0,0 +1,175 @@
+// ignore_for_file: avoid_void_async
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/data_models/onboarding_loader.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/pages/onboarding/onboarding_flow_navigator.dart';
+
+import '../../database/local_database.dart';
+
+class ConsentAnalytics extends StatelessWidget {
+  const ConsentAnalytics({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Size size = MediaQuery.of(context).size;
+    final LocalDatabase localDatabase = context.watch<LocalDatabase>();
+    final UserPreferences userPreferences = context.watch<UserPreferences>();
+    return Scaffold(
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          SizedBox(
+            height: size.height * 0.2,
+            width: size.width * 0.28,
+            child: Image.asset(
+              'assets/onboarding/data.png',
+              fit: BoxFit.contain,
+            ),
+          ),
+          Align(
+            alignment: Alignment.center,
+            child: Text(
+              'Send anonymous analytics',
+              maxLines: 1,
+              style: TextStyle(
+                  color: Colors.black,
+                  fontWeight: FontWeight.bold,
+                  fontSize: size.height * 0.025),
```

Comment:
Could you add an extra comma for the formatter?

Reply:
Solved